src/components/AdminPane/Manage/ManageChallenges/EditChallenge/Schemas/OSMCommitSchema.js

```diff
@@ -61,20 +72,38 @@ export const jsSchema = (intl, user, challengeData, extraErrors, options={}) =>
  * > the form configuration will help the RJSFFormFieldAdapter generate the
  * > proper markup
  */
-export const uiSchema = (intl, user, challengeData, extraErrors, options={}) => {
-  const isCollapsed = options.longForm && (options.collapsedGroups || []).indexOf(STEP_ID) !== -1
-  const toggleCollapsed = options.longForm && options.toggleCollapsed ? () => options.toggleCollapsed(STEP_ID) : undefined
+export const uiSchema = (
+  intl,
+  user,
+  challengeData,
+  extraErrors,
+  options = {}
+) => {
+  const isCollapsed =
+    options.longForm && (options.collapsedGroups || []).indexOf(STEP_ID) !== -1;
+  const toggleCollapsed =
+    options.longForm && options.toggleCollapsed
+      ? () => options.toggleCollapsed(STEP_ID)
+      : undefined;
+
+  const changesetUrl =
+    process.env.REACT_APP_CHANGESET_URL === "enabled" ? "changesetUrl" : null;
 
   const uiSchemaFields = {
     "ui:order": [
-      "checkinComment", "includeCheckinHashtag", "checkinSource",
+      "checkinComment",
+      "includeCheckinHashtag",
+      "checkinSource",
+      changesetUrl,
```

Comment:
Should this be quoted??

Reply:
this is a variable reference to the string value on line 90